data/starkscan.json

```
@@ -0,0 +1,34 @@
+{
+  "name": "STARKSCAN",
+  "description": "Next-generation StarkNet block explorer.",
+  "short_description": "StarkNet block explorer",
+  "tags": [],
+  "contracts": [],
+  "goerliContracts": [],
+  "audits": [],
+  "dotw": false,
+  "links": {
+    "website": "https://starkscan.co/?utm_source=dappland",
+    "careers": "jobs@aspect.co",
```

```suggestion
    "careers": "mailto:jobs@aspect.co",
```
Needs to include "mailto:" in order for it to work